drivers/dac/ad5791/ad5791.c

```diff
@@ -344,3 +344,73 @@ int32_t ad5791_setup(struct ad5791_dev *dev,
 
 	return status;
 }
+
+/***************************************************************************//**
+ * @brief SPI write to device using a mask.
+ *
+ * @param dev              - The device structure.
+ * @param register_address - Address of the register.
+ *                          Example:
+ *                          AD5791_REG_DAC          - DAC register
+ *                          AD5791_REG_CTRL         - Control register
+ *                          AD5791_REG_CLR_CODE     - Clearcode register
+ *                          AD5791_CMD_WR_SOFT_CTRL - Software control register
+ * @param mask	- The mask.
+ * @param value - The register data.
+ *
+ * @return 0 in case of success, negative error code otherwise.
+*******************************************************************************/
+int32_t ad5791_spi_write_mask(struct ad5791_dev *dev,
+			      uint8_t register_address,
+			      uint32_t mask,
+			      uint32_t value)
+{
+	int8_t status;
+	uint32_t reg_data;
+
+	status = ad5791_get_register_value(dev, register_address, &reg_data);
+	if (status < 0) {
```

Comment:
Checking nonzero value for status should be enough. 

No need for accolades.

Valid for other occurrences also.

Reply:
The existing API functions in the drivers also use this type of return check.
Do you consider changing them as well?

Reply:
Yes.